src/Symfony/Component/Messenger/Bridge/Redis/Transport/Connection.php

```diff
@@ -521,7 +521,7 @@ public function add(string $body, array $headers, int $delayInMs = 0): string
 
         try {
             if ($delayInMs > 0) { // the delay is <= 0 for queued messages
-                $id = uniqid('', true);
+                $id = bin2hex(random_bytes(4));
```

Comment:
Note that `random_bytes(4)` only has 32 bits of entropy, which is less than the previous call to `uniqid()` with `$more_entropy` set to `true`. This should use at least `random_bytes(8)` for 64 bits of entropy and ideally `random_bytes(16)` for 128 bits (the latter would result in a slightly longer output of 32 characters instead of 23 for uniqid).

Reply:
maybe we should use the initial suggestion from the issue and use a UUID instead (see #57872)